ctms/app.py

```diff
@@ -124,14 +124,14 @@ def get_contact_or_404(db: Session, email_id) -> ContactSchema:
 
 def all_ids(
     email_id: Optional[UUID] = None,
-    primary_email: Optional[EmailStr] = None,
+    primary_email: Optional[str] = None,
```

Comment:
I think we talked about this in other channels, but not in repo.

There are emails that fail pydantic / python-email-validator, such as `foo.@bar.com`. but are in the Salesforce / BigQuery list. I think the validator is correct, and these are invalid emails, potentially undeliverable, and including them in the our systems reduces the usefulness of the data.

There are also records with no emails, which seems even less useful than a invalid, perhaps mistyped email.

I think the right thing is to relax the constraints, like we're doing here, to get over the hump of the SFDC → Acoustic + CTMS transition. We also need an issue for the follow-on work of classifying the suspicious emails and blank values, and tightening up the acceptance of new data.

My hope was that we could be strict with new data and loose with existing data, but that is non-trivial work that has to be sacrificed to the deadline, and this is the right approach.

Reply:
Yeah, agreed. I think I feel much better now that this data is in the db successfully. We can start processing it shortly.